docs/usage/shoot_access.md

```diff
@@ -38,6 +38,48 @@ if err != nil {
 config = adminKubeconfigRequest.Status.Kubeconfig
 ```
 
+In Python you can use the native [`kubernetes` client](https://github.com/kubernetes-client/python) to create such a kubeconfig like this:
+
+```python
+# This script first loads an existing kubeconfig from your system, and then sends a request to the Gardener API to create a new kubeconfig for a shoot cluster. 
+# The received kubeconfig is then decoded and a new API client is created for interacting with the shoot cluster.
+
+import base64
+import json
+from kubernetes import client, config
+import yaml
+
+# Set configuration options
+shoot_name="my-shoot" # Name of the shoot
+project_namespace="my-namespace" # Namespace of the project
```

Comment:
The namespaces usually/always are prefixed with `garden-`. 
```suggestion
project_namespace="garden-my-namespace" # Namespace of the project
```

Reply:
good point, I updated it as well in the `kubectl` example where I stole the values from :)